build/vsts_build.yaml

```diff
@@ -529,33 +522,16 @@ jobs:
     # The Microsoft.DotNet.Build.Tasks.Feed package includes NuGet.Common 4.9.0.6 and a binding redirection in Microsoft.DotNet.Build.Tasks.Feed.dll.config but the binding redirection is not processed.
     # This would probably solve it:  https://github.com/Microsoft/msbuild/issues/1309
   - task: CmdLine@2
-    displayName: "Publish the packages to the .NET Core build asset registry (BAR)"
+    displayName: "Publish to the .NET Core build asset registry (BAR)"
     inputs:
-      script: dotnet msbuild ..\build\publish.proj /t:PublishPackagesToBuildAssetRegistry /p:NuGetClientNupkgsDirectoryPath=$(Build.Repository.LocalPath)\artifacts\$(NupkgOutputDir) /p:FeedUrl=$(DotNetFeedUrl) /p:BuildId=$(Build.BuildNumber) /p:RepoBranch=$(Build.SourceBranchName) /p:RepoCommit=$(Build.SourceVersion) /p:RepoUri=$(Build.Repository.Uri) /p:ManifestFilePath=$(ManifestFilePath) /p:AccountKey=$(dotnetfeed-storage-access-key-1)
+      script: dotnet msbuild ..\build\publish.proj /t:PublishToBuildAssetRegistry /p:NuGetClientNupkgsDirectoryPath=$(Build.Repository.LocalPath)\artifacts\$(NupkgOutputDir) /p:FeedUrl=$(DotNetFeedUrl) /p:BUILD_BUILDNUMBER=$(Build.BuildNumber) /p:BUILD_SOURCEBRANCH=$(Build.SourceBranchName) /p:BUILD_SOURCEVERSION=$(Build.SourceVersion) /p:BUILD_REPOSITORY_URI=$(Build.Repository.Uri) /p:ArtifactsLogDir=$(Build.Repository.LocalPath)\artifacts\manifests\ /p:MaestroApiEndpoint=$(MaestroApiEndpoint) /p:MaestroAccessToken=$(MaestroAccessToken)
       workingDirectory: cli
       failOnStderr: true
     env:
       DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
       DOTNET_MULTILEVEL_LOOKUP: true
     condition: " and(succeeded(),eq(variables['PublishArtifactsToDotNetBuildAssetRegistry'], 'true'), eq(variables['BuildRTM'], 'false')) "
 
-  - task: PowerShell@1
-    displayName: "Upload the build manifest file as a build artifact"
```

Comment:
Does the above task give us the manifest id now or?

Reply:
I couldn't figure out WHERE this task was copying the file to. I assumed it was the same place that JohnTortugo mentioned in a different comment is no longer needed. Since I copied the docs which was doing the copy in the MSBuild file, I assumed I could delete this.

When I did the insertions 2 weeks ago, I found the manifest ID (well, they call it a build id) in the build log, which is still there. Also, looking at a local copy of the manifest file, it doesn't contain the ID we need to use when using `darc add-build-to-channel ...`, so I don't think copying/uploading this file helps with that anyway.

Reply:
The manifest doesn't have an "ID"; it's just a description of the build assets and some other properties related to publishing. The Build ID (more precisely, the Maestro Build ID) is only shown in the logs from the `PushMetadataToBuildAssetRegistry` task. That's the ID that you'll use to promote/publish the build later with `darc add-build-to-channel`.

Reply:
@JohnTortugo is the Maestro build ID available as an output variable on the task, or is there some other way for us to programmatically get this value, other than parsing logs?  We'd eventually like to automate our insertions into shipping branches, using Azure Pipelines releases

Reply:
AFAIR it's only present as part of the logs, but that can be easily fixed. I think it's worth. Let me talk with my peers about it.